_includes/installation.html

```diff
@@ -234,36 +234,44 @@ <h3>Usage Example</h3>
+	<div class="main cli binary cplusplus linux odbc">
+		Linux 64-bit: <a href="https://artifacts.duckdb.org/latest/duckdb-binaries-linux.zip" target="_blank">https://artifacts.duckdb.org/latest/duckdb-binaries-linux.zip</a><br/><br/>
 
-		<p>Linux Build Artifacts are available from <a href="https://github.com/duckdb/duckdb/actions?query=branch%3Amaster+event%3Arepository_dispatch+workflow%3ALinuxRelease" target="_blank">the "LinuxRelease" CI runs</a></p>
+		$ wget https://artifacts.duckdb.org/latest/duckdb-binaries-linux.zip<br/>
+		$ curl -L https://artifacts.duckdb.org/latest/duckdb-binaries-linux.zip -o duckdb-binaries-linux.zip<br/><br/>
 
-		<p>Windows Build Artifacts are available from <a href="https://github.com/duckdb/duckdb/actions?query=branch%3Amaster+event%3Arepository_dispatch+workflow%3AWindows" target="_blank">the "Windows" CI runs</a></p>
-	</div>
+		Linux arm64/aarch64: <a href="https://artifacts.duckdb.org/latest/duckdb-binaries-linux-aarch64.zip" target="_blank">https://artifacts.duckdb.org/latest/duckdb-binaries-linux-aarch64.zip</a><br/><br/>
 
-	<div class="master cli binary macos">
-		MacOS Build Artifacts are available from <a href="https://github.com/duckdb/duckdb/actions?query=branch%3Amaster+event%3Arepository_dispatch+workflow%3AOSX" target="_blank">the "OSX" CI runs</a>
+		$ wget https://artifacts.duckdb.org/latest/duckdb-binaries-linux-aarch64.zip<br/>
+		$ curl -L https://artifacts.duckdb.org/latest/duckdb-binaries-linux-aarch64.zip -o duckdb-binaries-linux-aarch64.zip
 	</div>
 
-	<div class="master cli binary linux">
-		Linux Build Artifacts are available from <a href="https://github.com/duckdb/duckdb/actions?query=branch%3Amaster+event%3Arepository_dispatch+workflow%3ALinuxRelease" target="_blank">the "LinuxRelease" CI runs</a>
-	</div>
+	<div class="main cli binary cplusplus macos">
+		MacOS binaries (universal): <a href="https://artifacts.duckdb.org/latest/duckdb-binaries-osx.zip" target="_blank">https://artifacts.duckdb.org/latest/duckdb-binaries-osx.zip</a><br/><br/>
 
-	<div class="master cli binary win">
-		Windows Build Artifacts are available from <a href="https://github.com/duckdb/duckdb/actions?query=branch%3Amaster+event%3Arepository_dispatch+workflow%3AWindows" target="_blank">the "Windows" CI runs</a>
+		wget https://artifacts.duckdb.org/latest/duckdb-binaries-osx.zip<br/>
+		curl -L https://artifacts.duckdb.org/latest/duckdb-binaries-osx.zip -o duckdb-binaries-osx.zip
 	</div>
 
+	<div class="main cli binary cplusplus win odbc">
+		Win 64-bit: <a href="https://artifacts.duckdb.org/latest/duckdb-binaries-windows.zip" target="_blank">https://artifacts.duckdb.org/latest/duckdb-binaries-windows.zip</a><br/><br/>
+
+		wget https://artifacts.duckdb.org/latest/duckdb-binaries-windows.zip<br/>
+		curl -L https://artifacts.duckdb.org/latest/duckdb-binaries-windows.zip -o duckdb-binaries-windows.zip
```

Comment:
Curl isn't really used on Windows, except as the alias inside powershell (maybe just use invoke web request directly instead?)